### PR TITLE
fix: [SRM-10157]: Add white background while showing no data and error

### PR DESCRIPTION
**Pull request by praneshg239**
##### Summary:

Metrics

Pasted Graphic 1

### Logs screen

Log Screen Scenarios:

1. When both Radar chart and Logs list APIs are success don’t have data
image

2. While filtering through angle slider, if there is no data available in the logs list
image

3. When both API call fails (we have separate Retry buttons for two API calls)
image

4. If any one of the API call fails
image

image

##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

File: src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/DeploymentMetrics.tsx [hunk 1]
```diff
@@ -399,7 +400,13 @@ export function DeploymentMetrics(props: DeploymentMetricsProps): JSX.Element {
           </Layout.Horizontal>
         </Layout.Horizontal>
       </Layout.Horizontal>
-      <Container className={css.content}>{renderContent()}</Container>
+      <Container
+        className={cx(css.content, {
+          [css.whiteBackground]: (!currentViewData?.length && !loading) || getShouldShowError(error, shouldUpdateView)
```
Comment (deepesh-ui) [inline src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/DeploymentMetrics.tsx:405]: NIT: We can use defaultTo 

```suggestion
          [css.whiteBackground]: defaultTo(!currentViewData?.length && !loading,  getShouldShowError(error, shouldUpdateView))
```

**Review by deepesh-ui — APPROVED** [review]

File: src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/DeploymentMetrics.tsx (hunk 1 above)
Comment (harshil562) [inline src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/DeploymentMetrics.tsx:405]: Can we return this in a function and return the className ?

Comment (harshilbits) [inline src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/DeploymentMetrics.tsx:405]: Can we wrap this in a function with meaningful name and return the classname . As of now its not very clear that in which scenario we are showing whiteBackground?

Comment (praneshg239) [reply src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/DeploymentMetrics.tsx:405]: sure

Comment (shashidhargg) [reply src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/DeploymentMetrics.tsx:405]: No @deepesh-ui. `defaultTo` doesn't work like OR operator (||) And it only returns the default value if the first value is null, NaN, or undefined. Here we will get the first value as a boolean and if we get it as false, we want to check the second one.

**Comment by praneshg239** [comment]
trigger cypress

**Comment by praneshg239** [comment]
trigger cypress

**Review by harshilbits — DISMISSED** [review]

**Review by deepesh-ui — APPROVED** [review]

**Review by harshilbits — APPROVED** [review]

**Review by shashidhargg — APPROVED** [review]

**Review by harshilbits — APPROVED** [review]

**Comment by praneshg239** [comment]
trigger cypress

**Comment by praneshg239** [comment]
trigger cypress

**Pull request merged**